public/data/Kunal0007.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Kunal Patil",
+    "bio": "I am Kunal Patil currently pursuing a BTech degree in CS at PCCOE - Pimpri Chinchwad College Of Engineering, Pune. A Frontend Developer | Open-Source Enthusiast",
+    "avatar": "https://github.com/Kunal0007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/Kunal0007",
+        "icon": "github"
+      }
+    ]
+  }
+  
```

Comment:
needs a blank line ending at the end of the file (see the red mark by GitHub)

Reply:
Added blank line at the end of the file